spec/features/admin/merchant_items_spec.rb

```diff
@@ -90,14 +90,17 @@
 
     allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
 
-    visit "/admin/merchants/#{mike.id}/items"
+    visit "/admin/merchants/#{mike.id}/merchant_items"
     expect(page).to have_content(paper.name)
 
     within "#item-#{paper.id}" do
-      expect(page).to have_no_content("Activate Item")
+      expect(page).to have_content("Active?: true")
+      expect(page).to have_link("Deactivate Item")
       click_on("Deactivate Item")
+      expect(page).to have_content("Active?: false")
+      expect(page).to have_link("Activate Item")
       click_on("Activate Item")
-      expect(page).to have_content("Deactivate Item")
+      expect(page).to have_content("Active?: true")
```

Comment:
What'd you do to fix this?

Reply:
I moved resources to the bottom on the namespace, and added :update to resources :merchant_items, except